### ISSUE TITLE
Have the top screen photo/image on the DSi Menu theme be toggable on the settings.

### ISSUE THREAD

**Issue by Allinxter** [DS-Homebrew/TWiLightMenu#1836]
**Request description**
A user on the Discord server asked for help on how to disable the top screen photo/screenshot on the DSi theme. No such option exist, and though it is not something many uses have asked for, it wouldn't be a bad idea to have it be toggable, you know, for customization sake.

**Solution I'd like**
Add an option on the GUI settings tab of TWiLight Menu++ settings so you can toggle having a top screen image on or off. Additionally, since we're at it, an option to toggle the rotating cubes on the 3DS theme without having to toggle boxart would be nice as well, so you can chose to have the top screen clean on either theme if you want to.

**Alternatives considered**
Have a custom skin for DSi Theme that blanks the area and set RenderPhoto on the skin ini file to 0 (Sugested by Evie)). However, this would only work op this specific skin, and not others downloaded from the Skin site unless modified.

Have a valid image on the DSi Menu photos folder that's the color of the background. However this would leave the white outilne of the image still on the screen.

For the 3DS theme's rotating cubes you can disable it on the skin ini, but having it directly on the menu would be a more accessible option.

**Additional context**
I reckon this is quite the niche feature request, and I don't know how hard would it be to make this possible regardless of the menu skin. Still, I think it's worth the shot asking for it to be considered :D

**Comment by NightScript370** [comment]
The entire point of skins is to be customizable to avoid these types of questions.
You could easily switch all your skins over to not show the top screen button.

I'm in favor of closing.

**Comment by Allinxter** [comment]
Fair enough, considering it's quite a niche situation I don't think it would be hard to recommend anyone that encounters it to edit the ini file.

How about incluiding the steps to hide the photo under the "How do I show a custom picture on the top screen of the DSi theme?" section of the FAQ & Troubleshooting page of TWiLight Menu++ in the DS-Homebrew Wiki?

**Comment by NightScript370** [comment]
Alternative was chosen, so this isn't needed anymore.
Closed
